app/components/PillLabelBox.tsx

```diff
@@ -0,0 +1,45 @@
+import { tw } from '@helpwave/common/twind/index'
+
+export type PillLabelBoxProps = {
+  unscheduledCount: number,
+  inProgressCount: number,
+  doneCount: number
```

Comment:
the `Count` suffix is pretty much implied by context (namely the type `number` and the fact that there's 3 props)